staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss_cache.go

```diff
@@ -87,7 +87,10 @@ func (ss *scaleSet) newVMSSCache() (*timedCache, error) {
 		return localCache, nil
 	}
 
-	return newTimedcache(vmssTTL, getter)
+	if ss.Config.VmssCacheTTL == 0 {
+		return newTimedcache(time.Duration(vmssCacheTTLDefault)*time.Second, getter)
```

Comment:
prefer set `ss.Config.VmssCacheTTL` to the default value instead of return here, e.g. 

```go
if ss.Config.VmssCacheTTL == 0 {
    ss.Config.VmssCacheTTL = vmssCacheTTLDefault
}
```

Reply:
I agree with you, clearer than just return.